.golangci.yaml

```diff
@@ -33,6 +33,7 @@ linters:
     - goconst
     - golint
     - godot
+    - staticcheck
```

Comment:
I don't think we should disable this linter globally. I would rather see files excluded for this lint in the config, or the lint disabled inline where required with `// nolint:staticcheck` comments. The [docs detail the available options](https://golangci-lint.run/usage/false-positives/).

Reply:
Ya know, I tried that exact documentation when we were looking at custom linters awhile back it just does not appear to actually work. =/ I'm not sure if I was doing something wrong, but I remember there being a slew of issues and PR opens claiming that various "ignore this block of code" pragmas don't actually work.

Reply:
Weird, I haven't had trouble with it in the past. I've been using `//nolint`  comments in other projects 🤷‍♂️

Reply:
Yeah, alternatively, this could be PEBKAC so who knows.